core/src/main/java/org/bitcoinj/crypto/LazyECPoint.java

```diff
@@ -33,7 +33,7 @@ public class LazyECPoint {
     // If curve is set, bits is also set. If curve is unset, point is set and bits is unset. Point can be set along
```

Comment:
I see a couple of now outdated comments here and there, about the curve.

Reply:
Resolved and force-pushed.